website/docs/configuration/ort-yml.md

```diff
@@ -6,7 +6,9 @@ increased degree of automation and local configurations should only be done if t
 
 * [excludes](#excludes) - Mark [files, directories](#excluding-paths) or [package manager scopes](#excluding-scopes) as
   not included in released artifacts.
-* [curations](#curations) - Overwrite package metadata, set a concluded license or correct license findings.
+* [curations](#curations) - Override package metadata, set a concluded license or correct license findings. Needs to be
```

Comment:
Please change override back to overwrite as actions and decisions are overriden, information is overwritten.

Reply:
But doesn't "overwritten" imply that the original information is not available anymore? That's not the case here, and I actually deliberately chose "overridden" to reflect that.

Reply:
To me "overwrite" also sounds like the better fit here because we are replacing metadata (even if we document that it was overwritten by recording the applied curations), I checked the translations for "override" and they don't seem like a good fit to me.

Reply:
Ok, I'll drop that hunk.